src/main/java/no/nav/pto/veilarbportefolje/postgres/opensearch/PostgresOpensearchMapper.java

```diff
@@ -72,6 +74,14 @@ private void flettInnAktoerData(PostgresAktorIdEntity dataPaAktorId, Oppfolgings
         if (medDiffLogging) {
             loggDiff(dataPaAktorId, bruker);
         }
+        if (brukAvOppfolgingsdataPaPostgres(unleashService)) {
```

Comment:
what if brukAvOppfolgingsdataPaPostgres is false...its hard to find logic for that case